cdk.tf/vercel.json

```diff
@@ -190,6 +190,10 @@
     {
       "source": "/crash-reporting",
       "destination": "https://www.terraform.io/cdktf/telemetry#crash-reporting"
+    },
+    {
+      "source": "/.well-known/prebuilt-providers.json",
```

Comment:
any reason for the `.well-known`? I couldn't find the prebuilt-providers.json in the [wiki](https://en.wikipedia.org/wiki/Well-known_URI#:~:text=A%20well%2Dknown%20URI%20is,well%2Dknown%20locations%20across%20servers.)

Reply:
I tried to use that to suggest that this is depended on. E.g. the TF Registry uses this as well with something not found in the wiki `https://registry.terraform.io/.well-known/terraform.json` 😁

Reply:
ok, I see :)